### PR TITLE
Migrate to latest azure sdk

### PR DESCRIPTION
**Pull request by francoisferrand**
- Bump arsenal 8.1.75
- Migrate tests to new azure storage sdk

Issue: CLDSRV-305

### CODE REVIEW

**Review by nicolas2bert — APPROVED** [review]

File: package.json
Comment (ghost) [inline package.json]: Just a note not to forget to use a proper tag before merging

File: tests/functional/aws-node-sdk/test/multipleBackend/mpuParts/azurePutPart.js
Comment (ghost) [inline tests/functional/aws-node-sdk/test/multipleBackend/mpuParts/azurePutPart.js]: Why not:
```suggestion
        }, cb);
```

.. as the error message will stringify and maybe reduce the information carried by the error object?

**Review by ghost — COMMENTED** [review]
Will approve after a green build (please ping me if needed)

**Review by ghost — APPROVED** [review]
Now it's green :slightly_smiling_face:

**Pull request merged**
